include/wx/osx/cocoa/private/textimpl.h

```diff
@@ -60,6 +60,15 @@ public :
 
     virtual wxString GetStringValue() const override ;
     virtual void SetStringValue( const wxString &str) override ;
+    virtual wxString GetRTFValue() const override
+    {
+        wxFAIL_MSG("GetRTFValue() should only be used with multiline controls.");
+        return wxEmptyString;
+    };
```

Comment:
Minor, but this will result in warnings with gcc `-pedantic`:

```suggestion
    }
```

Reply:
Got it.